modules/mineapi/src/commands/list.ts

```diff
@@ -0,0 +1,26 @@
+// Import Types
+import { Command } from "@spookelton/wrapperHelpers/types";
+
+import { getStatus } from ".."
+import { helpHelper, strOut } from "@spookelton/wrapperHelpers/modul";
+
+export const list: Command = async (message) => {
+	const serverStatus = await getStatus();
+
+	const { maxPlayers, onlinePlayers, samplePlayers } = serverStatus;
+	const onlineMessage = samplePlayers?.length
+						  ? `:\n${samplePlayers.map(x => x.name).join(", ")}`
+						  : ".\nIt is quite lonely.";
+
+	const outputMessage = `There are ${onlinePlayers}/${maxPlayers} players online` + onlineMessage;
+
+	// TODO: Make better formatting.
+	// Includes perhaps showing more detailed userinfo, icon, etc.
+	// TODO: Do I have to do something to handle console/ingame output? Ideally there should be no output there, or
```

Comment:
Stroutput automatically formats a string for output on all types, so console/discord/ingame will be handled.